CHANGELOG.md

```diff
@@ -1,11 +1,52 @@
 # Changelog
 
-**THIS CHANGELOG HAS BEEN DEPRECATED**
+This changelog contains notable changes that make into each [`cardano-node` release](https://github.com/input-output-hk/cardano-node/releases),
+starting with version `8.0`. We also provide per-Haskell-package changelogs. Please see
+the `CHANGELOG.md` for each individual package for any changes relevant for developers.
 
-We have adopted per-Haskell-package changelogs. Please see the `CHANGELOG.md` for each
-individual package for any notable changes.
+If you are looking for the Ledger Releasing and Versioning Process then you can find it in
+[REALEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
```

Comment:
small typo  "REALEASING" -> "RELEASING" 
(though  "releasing" is a pretty cool idea hehe)

Reply:
Thank you. Fixed